kclvm/tools/src/LSP/src/to_lsp.rs

```diff
@@ -21,6 +21,16 @@ pub fn lsp_pos(pos: &KCLPos) -> Position {
     }
 }
 
+pub fn lsp_location(file_path: String, start: &KCLPos, end: &KCLPos) -> Location {
+    Location {
+        uri: Url::from_file_path(file_path).unwrap(),
```

Comment:
In the language server, we should try our best to avoid any form of panic function calls. Please adopt a defensive programming approach and add relevant logs if necessary.

Reply:
thanks, fixed